### PR TITLE
Review one

### PR DESCRIPTION
**Pull request by keza681**
## Portfolio part1
 In this pull request I made four commits actions: 
- Html page creation
- project styling
- Described the Readme file
- Fixed Linters errors

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+   <header>
+       <div class="container">
+        <div class="logo-container">
```
Comment (Meltrust) [inline Index.html:14]: - [x] Please kindly make this a link `<a>` tag, so the UX improves a lot for the user.

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <nav class="bars-container">
+            <a href=""><i class="fas fa-bars"></i></a>
+        </nav>
+       </div>
+   </header>
+   
+   <main class="main-body">
+    <section class="section">
+      <div class="paragraph">
+        <h1>I am Linda K. <br />Glad to see <br />you!</h1>
```
Comment (Meltrust) [inline Index.html]: - [x] Here, please kindly correct the font styling with the correct values for properties such as `font-weight`, `line-height`, and the `font-size`.  All the font values are stated on the Figma template.  That way, your project will be consistent 💪

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <p>
```
Comment (Meltrust) [inline Index.html:27]: - [x] Here, please kindly correct the font styling with the correct values for properties such as `font-weight`, `line-height`, and the `font-size`.  All the font values are stated on the Figma template.  That way, your project will be consistent 💪

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+          I am a software developer! I can help you build a product, feature
+          or website Look through some of my work and experience! if you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <div class="icons">
+          <div class="icons-title">
+            <h3>LET'S CONNECT</h3>
```
Comment (Meltrust) [inline Index.html:35]: - [x] Here, please kindly correct the font styling with the correct values for properties such as `font-weight`, `line-height`, and the `font-size`.  All the font values are stated on the Figma template.  That way, your project will be consistent 💪

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
```
Comment (Meltrust) [inline Index.html:11]: - [x]  Here is a summary in of your fonts that have styling errors.  Please kindly fix them so the project text looks a lot more consistent.

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @keza681, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
I can see you did some hard work, good job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The font styling throughout the project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1 +1,28 @@
...
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/linda-keza-a10150218/)
+
+#### Show your support
+
+Give a ⭐️ if you like this project!
+
+
+##### 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Hamzaoutdoors) [inline README.md:28]: [**OPTIONAL**]
- [x] I noticed your **MIT Licence** is a broken link. Please, be sure to make it as a link to your MIT LICENSE file after adding it to your root folder. Having a broken link in your README.md file is unprofessional.

File: Index.html [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+   <header>
+       <div class="container">
+        <div class="logo-container">
+            <p>Linda</p>
```
Comment (Hamzaoutdoors) [inline Index.html]: - [x]  You are very close to finishing your project @keza681, but this issue that @Meltrust pointed still remains on your code, t's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.

File: Index.html (hunk 2 above)
Comment (Hamzaoutdoors) [reply Index.html]: 💹 this issue well resolved

File: Index.html (hunk 3 above)
Comment (Hamzaoutdoors) [reply Index.html:27]: 💹 this issue well resolved

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @keza681, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by keza681** [comment]
## Additional information
Hi reviewer, I did my best to get my project done according to all the corrections I got from the previous reviewers.  So please consider that and the fact that this is my last try.
Kindly.

File: Index.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+   <header>
+       <div class="container">
+        <div class="logo-container">
+            <a href="#">Linda</a>
```
Comment (mateo951) [inline Index.html:15]: Addressed 👌

**Review by mateo951 — APPROVED** [review]
Hi @keza681,

You did an amazing job implementing all the adjustments that your previous reviewers suggested 🔥😎 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**